BatalanVlad/src/PaooGame/Game.java

```diff
@@ -15,26 +15,67 @@
 import java.awt.image.BufferStrategy;
 import java.util.ArrayList;
 
+
+/**
+ * The main instance of the game
+ */
 public class Game implements Runnable
```

Comment:
Class responsible for running an instance of the game. All class comments should start with "class responsible for"

Reply:
I left this as it is and I'll try to use for the next ones